drivers/espi/espi_npcx.c

```diff
@@ -295,9 +295,22 @@ static void espi_bus_cfg_update_isr(const struct device *dev)
 static void espi_bus_oob_rx_isr(const struct device *dev)
 {
 	struct espi_npcx_data *const data = dev->data;
+#if defined(CONFIG_ESPI_OOB_CHANNEL_RX_ASYNC)
+struct espi_reg *const inst = HAL_INSTANCE(dev);
```

Comment:
wrong indentation?

Reply:
Oops. Thanks for pointing out this, Will fix it later.

Reply:
Done.